web/src/components/atoms/CustomTag/index.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import { CSSProperties } from "react";
+
+type CustomTagProps = {
+  value?: number | string;
+  color?: CSSProperties["color"];
+};
+
+const CustomTag: React.FC<CustomTagProps> = ({ value, color }) => {
+  const defaultColor = "#87e8de";
+  const defaultValue = "";
```

Comment:
Either just use these values directly where they are used, or move them outside of the component as they won't ever change